File: Python/program/Sivitas.py
# import class file
from Human import Human

# class declaration


class Sivitas (Human):

    # atribut private
    __asal_univ = ""
    __email_edu = ""

   # konstruktor
    def __init__(self, asal_univ="", email_edu=""):
        self.__asal_univ = asal_univ
        self.__email_edu = email_edu

    # Getter and Setter

    def get_asal_univ(self):
        return self.__asal_univ

    def set_asal_univ(self, asal_univ):
        self.__asal_univ = asal_univ

    def get_email_edu(self):
        return self.__email_edu

    def set_email_edu(self, email_edu):
        self.__email_edu = email_edu

File: Java/program/Sivitas.java
// deklarasi kelas Sivitas
// kelas sivitas adalah child dari kelas human
public class Sivitas extends Human {
    // atribut private
    private String asal_universitas, email_edu;

    /* konstruktor */

    // konstruktor tanpa parameter
    public Sivitas() {
        // set isi atribut secara default
        this.asal_universitas = "";
        this.email_edu = "";
    }

    // konstruktor dengan parameter
    public Sivitas(String asal_universitas, String email_edu) {
        // set isi atribut secara default
        this.asal_universitas = asal_universitas;
        this.email_edu = email_edu;
    }
    /* Getter dan Setter */

    // Get asal_universitas
    public String getAsal_universitas() {
        return this.asal_universitas;
    }

    // set asal_universitas
    public void setAsal_universitas(String asal_universitas) {
        this.asal_universitas = asal_universitas;
    }

    // Get email_edu
    public String getEmail_edu() {
        return this.email_edu;
    }

    // set email_edu
    public void setEmail_edu(String email_edu) {
        this.email_edu = email_edu;
    }
}
